core/modules/widgets/button.js

```diff
@@ -200,7 +189,6 @@ ButtonWidget.prototype.execute = function() {
 	this.setTo = this.getAttribute("setTo");
 	this.popup = this.getAttribute("popup");
 	this.hover = this.getAttribute("hover");
-	this["class"] = this.getAttribute("class","");
```

Comment:
I would like to keep this line, so it's clear that we didn't forget about the value. .. Later in your code you could use `this.class` instead of `this.getAttribute`

Reply:
> Later in your code you could use `this.class` instead of `this.getAttribute`

That will not work, as `this.class` will not be updated on refresh, which is why we need to use `this.getAttribute()` again in the refresh handler.

Reply:
If really necessary I would favour adding a comment here rather than redundant code.

Reply:
I'm OK with at comment.